content/en/docs/concepts/architecture/garbage-collection.md

```diff
@@ -95,8 +95,10 @@ to learn more.
 ### Background cascading deletion {#background-deletion}
 
 In background cascading deletion, the Kubernetes API server deletes the owner
-object immediately and the controller cleans up the dependent objects in
-the background. By default, Kubernetes uses background cascading deletion unless
+object immediately and the controller (like the custom or default controller)
```

Comment:
Does this controller have a name? Default controller is a bit vague 

Thank you for the pr

Reply:
I believe this is the [garbage collector](https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/garbagecollector/graph_builder.go) controller.

Reply:
I could use the below instead.
```suggestion
object immediately and the garbage collector controller (custom or default)
```